src/components/selectboxes/SelectBoxes.js

```diff
@@ -110,6 +112,41 @@ export default class SelectBoxesComponent extends RadioComponent {
     this.updateValue(flags);
   }
 
+  onChange(flags, fromRoot) {
```

Comment:
This code isn't needed... the validator should get triggered on change already and all the logic exists in the validators and should not be duplicated here.